graph/pull.go

```diff
@@ -40,6 +59,10 @@ func (s *TagStore) Pull(image string, tag string, imagePullConfig *ImagePullConf
 		return err
 	}
 
+	if err := validateRepoNotScratch(repoInfo.RemoteName); err != nil {
```

Comment:
please use `validateRepoName(repoInfo.LocalName)` instead. Defining a separate `validateRepoNotScratch()` function is not necessary.